src/client.js

```diff
@@ -79,6 +80,27 @@ module.exports = class Client {
     })
   }
 
+  //maintains pool of static endpoints with auto-batching enabled
```

Comment:
Nit: All comments should follow the schema of a grammatically correct sentence:

```suggestion
// Maintains a pool of static endpoints with auto-batching enabled
```

(Here and others)